change/@fluentui-react-overflow-a2d80786-56a5-418d-8c9e-b6cbb8916016.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33031&commitId=8562ff9a2a426bef67b75fb8a512a261548d900f&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|7| Changed|

</details>


  <details><summary><b>Drawer</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|984| Changed|
|Drawer.overlay drawer full.chromium.png|1154| Changed|
|Drawer.Full Overlay RTL.chromium.png|3315| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>